src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Promotion.xml

```diff
@@ -136,7 +136,7 @@ Example configuration for `order_fixed_discount` action type:
         <subresourceOperations>
             <subresourceOperation name="coupons_get_subresource">
                 <attribute name="method">GET</attribute>
-                <attribute name="path">/admin/promotions/{code}/promotion-coupons</attribute>
+                <attribute name="path">/admin/promotions/{code}/coupons</attribute>
```

Comment:
Let's add a note about this change to the `UPGRADE-API` file. It's a pretty significant change that the endpoint has been changed 😅.

Reply:
This is a new endpoint that I added in a previous PR :)